packages/nextra/src/compile.ts

```diff
@@ -124,24 +121,17 @@ export async function compileMdx(
 
   try {
     const vFile = await compiler.process(
-      filePath ? { value: source, path: filePath } : source
+      filePath ? { value: content, path: filePath } : content
     )
-    let result = String(vFile).replace('export default MDXContent;', '')
 
     const headingMeta = compiler.data('headingMeta') as Pick<
       PageOpts,
       'headings' | 'hasJsxInH1' | 'title'
     >
-    if (headingMeta.title) {
-      result += `\nconst __nextra_title__ = ${JSON.stringify(
-        headingMeta.title
-      )}`
-    }
```

Comment:
since we no longer use the previous plugin, this code is no longer needed we can get `title` (`headingMeta.title`) in loader